src/FakeItEasy.Analyzer/UnusedReturnValueAnalyzer.cs

```diff
@@ -68,10 +68,9 @@ private static IMethodSymbol GetCalledMethodSymbol(InvocationExpressionSyntax ca
 
         private static string GetDiagnosticName(IMethodSymbol methodSymbol)
         {
-            var attribute = methodSymbol.GetAttributes()
-                .FirstOrDefault(a => a.AttributeClass.GetFullName() == "FakeItEasy.Analysis.MustUseReturnValueAttribute");
-
-            return attribute?.ConstructorArguments.FirstOrDefault().Value as string;
+            return methodSymbol.GetAttributes()
+                .FirstOrDefault(a => a.AttributeClass.GetFullName() == "FakeItEasy.Analysis.MustUseReturnValueAttribute")
+                ?.ConstructorArguments.FirstOrDefault().Value as string;

```

Comment:
https://stylecop.codeplex.com/workitem/7722 has been fixed in StyleCop.MSBuild 4.7.53.0